webapp-django/crashstats/tokens/static/tokens/js/home.js

```diff
@@ -1,14 +1,29 @@
 $(function() {
-    $('p.code button').click(function(event) {
+
+    'use strict';
+
+    $('.token p.code[data-key]').each(function() {
+        var p = $(this);
+        p.prepend(
+            $('<code>')
+            .addClass('truncated')
+            .on('click')
+            .text(p.data('key').substr(0, 12) + '…')
+        ).prepend(
+            $('<code>')
```

Comment:
I never know which is better. Perhaps it's just personal taste but I like the `.addClass()` and the `.text()` APIs separately. Choices are annoying. If you don't mind, I'll keep this style.